### ISSUE TITLE
Unable to reproduce same mAP with kitti pretrained tf1 model on Kitti dataset

### ISSUE THREAD

**Issue by mohsinmx** [tensorflow/models#10376]
- [ ] I am using the TensorFlow 1.14
- [ ] I checked to make sure that this issue has not already been filed.

## 1. The entire URL of the file you are using

https://github.com/tensorflow/models/tree/master/research/...

Downloaded Kitti pretrained model from 
http://download.tensorflow.org/models/object_detection/faster_rcnn_resnet101_kitti_2018_01_28.tar.gz

## 2. Describe the bug
When evaluation is run on test set of Kitti dataset, mAP for IoU=0.5 is 39%, instead of 87%..

## 4. Expected behavior

Should achieve mAP atleast near to 80%.

## 6. System information

- OS Platform and Distribution : Linux Ubuntu 20.04
- TensorFlow installed from pip
- TensorFlow version : 1.14
- Python version: 3.7
- CUDA/cuDNN version: not installed 
- GPU model and memory: not available

**Comment by kumariko** [comment]
@mohsinmx As you are using an older version of tensorflow(1.x) which is officially considered as end of life. We recommend that you upgrade to 2.6 or later version and let us know if the issue still persists in newer versions. Thanks!

**Comment by mohsinmx** [comment]
@kumariko , it is a tf1 model. How to use a tf1 model in tf2?

**Comment by Petros626** [comment]
> @kumariko , it is a tf1 model. Can we use a tf1 model in tf2 for training purpose?

I would say NO, but if someone knows it better..
